src/common/config.h

```diff
@@ -634,6 +638,133 @@ class Config {
     return ul_tx_f_data_files_;
   }
 
+  inline void TryEnqueueLogStatsMaster(moodycamel::ConcurrentQueue<EventData>* mc_queue,
+                                       moodycamel::ProducerToken* producer_token, 
+                                       const EventData& event,
+                                       size_t frame_id,
+                                       size_t symbol_id) {
+    size_t enqueue_start_tsc_ = 0;
+    size_t enqueue_end_tsc_ = 0;
+    if (frame_id == this->frame_to_profile_) {
+      enqueue_start_tsc_ = GetTime::WorkerRdtsc();
+    }
+    TryEnqueueFallback(mc_queue,
+                       producer_token,
+                       event);
+    if (frame_id == this->frame_to_profile_) {
+      enqueue_end_tsc_ = GetTime::WorkerRdtsc();
+      enqueue_stats_[symbol_id][enqueue_stats_id_.at(symbol_id)].tsc_start_ =
```

Comment:
TryEnqueueLogStatsMaster and the other function TryEnqueueLogStatsMaster look almost the same.  Remove the copy / paste code and call into a common function (or have 1 call the other)